component/Applaud/AllAplaud.jsx

```diff
@@ -1,22 +1,34 @@
-import { Col, Row } from "antd";
+import { Col, DatePicker, Row, Timeline } from "antd";
 import React, { useEffect, useState } from "react";
 import Image from "next/image";
 import User1 from "../../assets/images/User1.png";
 import { ApplaudGiven, ApplaudIconSmall } from "../../assets/Icon/icons";
+import { ClockCircleOutlined } from "@ant-design/icons";
+import moment from "moment";
 
 function AllAplaud({ user }) {
   const [allApplaud, setAllApplaud] = useState([]);
+  const [allOrgApplaud, setAllOrgApplaud] = useState([]);
+
+  const [currentMonth, setCurrentMonth] = useState({
+    lte: moment().endOf("month"),
+    gte: moment().startOf("month"),
+  });
 
   async function fetchApplaudData() {
     setAllApplaud([]);
-    if (user?.id) {
-      await fetch("/api/applaud/all/" + user.id, {
-        method: "GET",
+    if (user.id) {
```

Comment:
Remove check if user dosn't exist , dashboard should not be visible and api calls shoulbe be central